Examples/SyncUps/SyncUpsTests/AppFeatureTests.swift

```diff
@@ -17,24 +17,22 @@ final class AppFeatureTests: XCTestCase {
       )
     }
 
-    await store.send(.path(.push(id: 0, state: .detail(SyncUpDetail.State(syncUp: syncUp))))) {
+    await store.send(\.path[id:0], .detail(SyncUpDetail.State(syncUp: syncUp))) {
```

Comment:
I added a case path for pushing/popping based off state vs. `nil`. Here's pushing...